tests/framework/CodeInterfaceTests/CobraTF/CobraTFinterfaceTest2/testDummyStep_MC/1/small_3x3rod_HFP.ctf.out

```diff
@@ -0,0 +1,1132 @@
+h********************************* input file listing *********************************
```

Comment:
are we OK with putting this input file in the repo?
are we OK from an export control point of view?

Reply:
I checked it with NCSU team (Paolo Balestra, Maria Avramova) and there's no export control issue.